src/plugins/audiobridge-plugin.js

```diff
@@ -52,6 +54,7 @@ const PLUGIN_EVENT = {
   ALLOWED: 'audiobridge_allowed',
   SUCCESS: 'audiobridge_success',
   ERROR: 'audiobridge_error',
+  ROOM_MUTED: 'audiobridge_room_muted'
```

Comment:
As a convention, I'd prefer having `SUCCESS` and `ERROR` as last events, so could you please move `ROOM_MUTED` two positions up ?